lib/storage/s3.js

```diff
@@ -55,6 +55,9 @@ module.exports = function() {
       client = new AWS.S3(options);
       if (options.contentTypes) {
         contentTypes = options.contentTypes;
+        if (!contentTypes.svg) {
+          contentTypes.svg = 'image/svg+xml';
```

Comment:
I'm rusty, but isn't this the case where the user is explicitly passing the content types *they* want? Does it make sense to force in another one in that case?

Reply:
Yeah, I think you're right the user is passing content types themselves here. It's _probably_ safe to assume that if they're doing this they can handle the SVG issue. 

Maybe a better option though would be to merge the user input with `contentTypes.js`? That way nothing is missed, but any changes the user wants are accounted for.